spec/controllers/api/diagnoses_controller_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Api::DiagnosesController, type: :controller do
+  login_user
+
+  let(:visit) { create :visit, advisor: current_user }
+  let(:content) { Faker::Lorem.paragraph }
+  let(:diagnosis) { create :diagnosis, visit: visit, content: content }
```

Comment:
I would say `let(:content)` is useless here.
Prefer : `let(:diagnosis) { create :diagnosis, visit: visit, content: Faker::Lorem.paragraph }`